metagraph/src/graph/alignment/aligner_alignment.hpp

```diff
@@ -49,7 +57,8 @@ class Alignment {
               bool orientation = false,
               size_t offset = 0);
 
-    void append(Alignment&& other);
+    // return true if the internal labels were modified
+    bool append(Alignment&& other);
```

Comment:
Does it only add labels or may remove some as well?

Reply:
This concatenates two alignments and intersects their labels. If the alignments have coordinates, then it makes sure that the two alignments have adjacent index ranges

Reply:
Sorry, my question was about this comment: "... internal labels were modified"

Reply:
ah, ok. So I'll expand the comment to describe exactly what this function is doing. It returns true if some of the labels or coordinates had to be discarded during the append.